instagram-app/src/components/PostContainer/PostsPage.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PostContainer from './PostContainer'
+import SearchBar from '../SearchBar/SearchBar'
+
+import dummyData from '../../dummy-data';
+
+
+
+
+class PostsPage extends React.Component {
+  constructor(){
+    super();
+    this.state = {
+      dummy: [],
+      searchPosts: []
+    };
+
+  }
+  componentDidMount(){
+    this.setState({dummy:dummyData});
+    this.setState({searchPosts:dummyData});
+  }
```

Comment:
You don't have to make multiple calls to `.this.setState` - you can combine them like so:
```
this.setState({
  dummy: dummyData,
  searchPosts: dummyData
});
```

The syntax is like definition multiple properties for an object.